inventory/sample.aws.example.com.d/inventory/group_vars/OSEv3.yml

```diff
@@ -35,8 +35,9 @@ osm_cockpit_plugins:
 - 'cockpit-kubernetes'
 
 # OpenShift FQDNs, DNS, App domain specific configurations
+openshift_master_cluster_method: native
 openshift_master_default_subdomain: "apps.{{ env_id }}.{{ dns_domain }}"
-openshift_master_cluster_hostname: "master-0.{{ env_id }}.{{ dns_domain }}"
+openshift_master_cluster_hostname: "console.internal.{{ env_id }}.{{ dns_domain }}"
```

Comment:
Unless I'm missing it, it's not clear where you are setting the `internal` portion of the DNS zone(?)

Reply:
Yes, this is handle by the cloudformation template:

```
- Name: "{{ openshift_master_cluster_hostname }}."
        Type: A
        AliasTarget:
          HostedZoneId: !GetAtt MasterExtElb.CanonicalHostedZoneNameID
          DNSName: !GetAtt MasterExtElb.CanonicalHostedZoneName
```